components/alert/README.md

```diff
@@ -1,63 +1,120 @@
 # Alerts
+Alerts communicate critical information relating to the state of the system and the users’ work flow.
```

Comment:
I'm thinking this should be `user's` work flow if I'm remembering grammar correctly - we user it below in `Inline Alert` but as `user's` so we probably just want to be consistent.